db/seeds/08_ambient_sounds.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO ambient_sounds (type)
+VALUES ('nature'),
```

Comment:
can we include 'none' as the first/default value for sounds?

Reply:
same for sounds. so if a user just picks a photo, then their theme will only have a photo_id and null for animation_id and ambient_sound_id